addons/mrp/report/mrp_report_bom_structure.py

```diff
@@ -30,7 +30,7 @@ def _compute_current_production_capacity(self, bom_data):
             if comp['product'].type != 'product' or float_is_zero(comp['base_bom_line_qty'], precision_digits=comp['uom'].rounding):
                 continue
             components_qty_to_produce[comp['product_id']] += comp['base_bom_line_qty']
-            components_qty_available[comp['product_id']] = comp['quantity_available']
+            components_qty_available[comp['product_id']] = comp['free_to_manufacture_qty']
```

Comment:
```suggestion
            components_qty_available[comp['product_id']] = comp['quantity_available']
```
It was already good like that if I'm not wrong on my other suggestions